interactions/client/models/command.py

```diff
@@ -862,7 +860,8 @@ async def __call(
             if has_args:  # foo(ctx, ..., *args, ..., **kwargs) OR foo(ctx, *args, ...)
                 return await _coro(
                     ctx,
-                    *(kwargs[opt] for opt in par_opts if opt in kwargs),  # pos before *args
+                    # pos before *args
+                    *(kwargs[opt] for opt in par_opts if opt in kwargs),
```

Comment:
?

Reply:
Oops looks like my IDE just reformatted that
I'll fix that